tests/utils/dockercli/containers.go

```diff
@@ -122,7 +124,10 @@ func GetVolumeStatus(hostName, volumeName string) (map[string]string, error) {
 	}
 
 	status := make(map[string]string)
-	val := strings.Fields(out)
+	val := strings.SplitAfter(out, volmPropertyDelimiter)
```

Comment:
will this not work https://golang.org/pkg/strings/#Split
with sep as volmPropertyDelimiter